app/models/library_entry.rb

```diff
@@ -99,9 +99,15 @@ def activity
   end
 
   before_save do
-    if status_changed? && status == :completed && media.progress_cap
+    if status_changed? && status == 'completed' && media.progress_limit
```

Comment:
I remembered that you can actually just do `&& completed?` due to the way enums work!